cirq-google/cirq_google/calibration/engine_simulator.py

```diff
@@ -304,6 +305,45 @@ def create_from_characterizations_sqrt_iswap(
             ideal_when_missing_parameter=ideal_when_missing_parameter,
         )
 
+    @classmethod
+    def create_from_gate_dict(
```

Comment:
The other method is called `create_from_dictionary_sqrt_iswap`, maybe `create_from_dictionary` would be more consistent with that?

Could you also move this method up, closer to the original one?

Reply:
Done.